python/Ganga/Core/GangaRepository/VStreamer.py

```diff
@@ -76,17 +76,21 @@ def to_file(j, fobj=None, ignore_subs=''):
 # * AssertionError (corruption: multiple objects in <root>...</root>
 # * Exception (probably corrupted data problem)
 
-def _raw_from_file(f):
+def _raw_from_file(f, obj):
     # logger.debug('----------------------------')
     ###logger.debug('Parsing file: %s',f.name)
     xml_content = f.read()
-    obj, errors = Loader().parse(xml_content)
+    xml_lines = xml_content.splitlines()
+    for line in xml_lines:
+        line = line.lstrip()
+    xml_content = '\n'.join(xml_lines)
+    obj, errors = Loader(obj).parse(xml_content)
```

Comment:
The thing is we can only ever pre-create the root object as we don't know what nested objects there are in the tree. There is almost always going to be lots of nested objects but there will only ever be one root object in a file.